partiql-eval/src/main/java/org/partiql/eval/value/CharValue.java

```diff
@@ -0,0 +1,34 @@
+package org.partiql.eval.value;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.value.PartiQLValueType;
+
+/**
+ * This shall always be package-private (internal).
+ */
+class CharValue implements PQLValue {
```

Comment:
Similar comment, but seems we don't need Char and String considering their identical implementations.